specs/commandline/blob.md

```diff
@@ -0,0 +1,501 @@
+  -h,  --help                       help for sign
+```
+
+### notation blob inspect
+
+```text
+Inspect a signature associated with a blob
+
+Usage:
+  notation blob inspect [flags] <signature_path>
+
+Flags:
+  -o, --output string         output format, options: 'json', 'text' (default "text")
+  -d, --debug                 debug mode
+  -v, --verbose               verbose mode
+  -h, --help                  help for inspect
+```
+
+### notation blob policy
+
+```text
+Manage trust policy configuration for arbitrary blob signature verification.
+
+Usage:
+  notation blob policy [command]
+
+Available Commands:
+  import    import trust policy configuration from a JSON file
+  show      show trust policy configuration
+
+Flags:
+  -h, --help   help for policy
+```
+
+### notation blob policy import
+
+```text
+Import blob trust policy configuration from a JSON file
+
+Usage:
+  notation blob policy import [flags] <file_path>
+
+Flags:    
+      --force     override the existing trust policy configuration, never prompt
+  -h, --help      help for import
+```
+
+### notation blob policy show
+
+```text
+Show blob trust policy configuration
+
+Usage:
+  notation blob policy show [flags]
+
+Flags:
+  -h, --help      help for show
+```
+
+### notation blob verify
+
+```text
+Verify a signature associated with a blob
+
+Usage:
+  notation blob verify [flags] --signature <signature_path> <blob_path>
+
+Flags:
+      --signature string      location of the blob signature file
+      --media-type string     optional media type of the blob to verify
+      --policy-name string   optional policy name to verify against. If not provided, notation verifies against the global policy if it exists.
+  -m, --user-metadata stringArray   user defined {key}={value} pairs that must be present in the signature for successful verification if provided
+      --plugin-config stringArray   {key}={value} pairs that are passed as it is to a plugin, if the verification is associated with a verification plugin, refer plugin documentation to set appropriate values
+  -o, --output string         output format, options: 'json', 'text' (default "text")
+  -d, --debug                 debug mode
+  -v, --verbose               verbose mode
+  -h, --help                  help for inspect
+```
+
+## Usage
+
+## Produce blob signatures
+
+### Sign a blob by adding a new key
+
+```shell
+# Prerequisites:
+# - A signing plugin is installed. See plugin documentation (https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md) for more details.
+# - Configure the signing plugin as instructed by plugin vendor.
+
+# Add a default signing key referencing the remote key identifier, and the plugin associated with it.
+notation key add --default --name <key_name> --plugin <plugin_name> --id <remote_key_id>
+
+# sign a blob
+notation blob sign /tmp/my-blob.bin
+```
+
+An example for a successful signing:
+
+```console
+$ notation blob sign /tmp/my-blob.bin
```

Comment:
I'm wondering why all examples are signing a file with an _absolute_ path? Can `notation blob sign/verify` take a _relative_ path?

Reply:
Relative paths should be supported. Its just that there were not called out explicitly. Added couple of examples for sign and verify.